src/js/edu-benefits/containers/SubmitPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import SubmitMessage from '../components/SubmitMessage';
+
+class SubmitPage extends React.Component {
+  render() {
+		// TODO: shared address component?
+    const address = (
```

Comment:
I think we'd want to get this from the api eventually, too? I think region depends on some data in the form.